src/backend/launcher.ts

```diff
@@ -803,7 +803,8 @@ export async function verifyWinePrefix(
     return { res: { stdout: '', stderr: '' }, updated: false }
   }
 
-  if (!existsSync(winePrefix) && !(await isUmuSupported(wineVersion.type))) {
+  let newPrefix = !existsSync(winePrefix)
```

Comment:
this won't work as well as creating a file because a user could pick an existing prefix which this would return false for